README.md

```diff
@@ -83,14 +83,21 @@ $ wrangler publish
 
 ### 🔧 `config`
 
-  Configure your global Cloudflare user. This is an interactive command that will prompt you for your email and API key:
+  Configure your global Cloudflare user. This is an interactive command that will prompt you for your API token:
 
   ```bash
   wrangler config
+  Enter API token:
+  superlongapitoken
+  ```
+
+  You can also provide your email and global API key (this is not recommended for security reasons):
+  ```bash
+  wrangler config --api-key
   Enter email:
   testuser@example.com
-  Enter api key:
-  ...
+  Enter global API key:
+  superlongapikey
   ```
 
   You can also [use environment variables](https://developers.cloudflare.com/workers/tooling/wrangler/configuration/) to configure these values.
```

Comment:
for later, and perhaps i should make an issue for it: we should be clear about precedence.

e.g. (strawman): if you have both a config file and env vars, env vars take precedence; if you have all three env vars defined, api token takes precedence.

Reply:
for sure! I think we'll discuss this behavior in next week's wrangler maintainer meeting and make sure we 100% want it before detailing it in the readme.

Reply:
yah again is strawman suggestion, but the key point is to document the decision. i can make an issue.

Reply:
linked: https://github.com/cloudflare/wrangler/issues/834